users/models.py

```diff
@@ -188,6 +186,19 @@ def get_project_ordered_by_last_report_creation_date(self) -> QuerySet:
             )
         ).order_by("-last_report_creation_date")
 
+    @staticmethod
+    def __parse_work_hours_and_percentage_from_queryset_to_a_dict(
```

Comment:
The current unit tests in `test_mixins.py` already cover this portion of code. Therefore I didn't add any additional unit tests for this function alone.